app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  has_many :cities
+  has_many :reviews, through: :cities
+  has_many :votes, through: :reviews
   mount_uploader :avatar, AvatarUploader
   # Include default devise modules. Others available are:
```

Comment:
The second two has_many associations should not need the `through` keywords. 
E.g. there is a direct relationship between a user and a review, so the through keyword is not needed